src/core/lv2/Lv2Manager.cpp

```diff
@@ -133,6 +133,26 @@ const std::set<const char*, Lv2Manager::CmpStr> Lv2Manager::pluginBlacklist =
 	"urn:juced:DrumSynth"
 };
 
+const std::set<const char*, Lv2Manager::CmpStr> Lv2Manager::pluginBlacklistBuffersizeLessThan32 =
+{
+	"http://moddevices.com/plugins/mod-devel/2Voices",
+	"http://moddevices.com/plugins/mod-devel/Capo",
+	"http://moddevices.com/plugins/mod-devel/Drop",
+	"http://moddevices.com/plugins/mod-devel/Harmonizer",
+	"http://moddevices.com/plugins/mod-devel/Harmonizer2",
+	"http://moddevices.com/plugins/mod-devel/HarmonizerCS",
+	"http://moddevices.com/plugins/mod-devel/SuperCapo",
+	"http://moddevices.com/plugins/mod-devel/SuperWhammy",
+	"http://moddevices.com/plugins/mod-devel/GX2Voices",
```

Comment:
Typo, my bad. It's 'Gx' and not 'GX'. So Gx2Voices, etc..

Reply:
Thansk, fixed.